src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 0;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    color: $main-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin-bottom: 18px;
+    color: $second-color;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    color: $main-color;
+    text-align: right;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__cost {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    color: $second-color;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__amount {
+    color: $main-color;
+    text-align: right;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__btn {
+    margin-bottom: 16px;
+    width: 100%;
+    height: 40px;
+    border: none;
+    border-radius: 5px;
+    background: $btn-color;
+    font-family: Roboto, sans-serif;
+    color: $white-color;
+    text-align: center;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    cursor: pointer;
+
+    &:hover {
```

Comment:
No need to re-declare border-radius on hover, it's already defined in the base class.